source/index.js

```diff
@@ -13,6 +12,8 @@ const DEFAULT_OPTIONS = {
 const EMPTY_LINE = /^\s*$/;
 const VARIABLE_DEFINITION = /^([a-zA-Z]+)=(.+)$/;
 
+const DOMAIN_EXP = /\b((?=[a-z0-9-]{1,63}\.)(xn--)?[a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,63}\b/i;
```

Comment:
I think it's a bad idea to have this hardcoded in this library. Domain validation _should_ be the responsibility of another library or tool **dedicated** to that purpose.

The previous `domain-regex` was indeed case-sensitive making it flawed, but [`is-domain-name`](https://www.npmjs.com/package/is-domain-name) looks to work fine. I would request that you use that instead.

Reply:
The previous domain-regex doesn't recognize capitalized domain names and is not under maintenance for more than 3 years. To keep dependency just for one string is not good idea too. I just used it as a base and added case insensitive flag /i.

Reply:
What do you think about https://github.com/miguelmota/is-valid-domain ?

Reply:
> domain-regex doesn't recognize capitalized domain names

Yep, had already agreed with you in my first comment.

> To keep dependency just for one string is not good idea

I disagree.. The domain is a very important part of what ads.txt is. Arbitrarily saying a dependency is not important based upon how often it is used, rather than what it is used for, is not a great basis for removing said dependencies.

> What do you think about miguelmota/is-valid-domain 

This looks good.. I would accept changes that use this instead 👍